opentelemetry-api/tests/util/test__importlib_metadata.py

```diff
@@ -24,11 +28,67 @@ def test_entry_points(self):
         self.assertIsInstance(
             next(
                 iter(
-                    entry_points(
```

Comment:
So now entry_points returns a list?
if so, Consider removing the next(iter(entry_points)) and just verify the len of the list and get list[0]
it will be more readable, and you don't get a chance to get a StopIteration exception

Reply:
Both approaches are equivalent, would rather not change that since it is outside the scope of this PR.